src/renderer/lib/api.ts

```diff
@@ -179,6 +179,10 @@ export type Part = z.infer<typeof Part>;
 const Project = z.object({
   label: z.string(),
   value: z.string(),
+  repoUrl: z
```

Comment:
maybe `z.string().default("")` (?)

Reply:
default is to replace `undefined` only, not null